README.md

```diff
@@ -40,57 +39,55 @@ In the end, we will summarize our discussion with links to PCA alternatives.
 
 # Steps
 
-This Code Pattern consists of following activities:
+This code pattern consists of following activities:
 
 * [Run a Jupyter notebook in the IBM Watson Studio](#run-a-jupyter-notebook-in-the-ibm-watson-studio).
 * [Deep Dive into Principal Component Analysis](#deep-dive-into-principal-component-analysis).
 
 ## Run a Jupyter notebook in the IBM Watson Studio
 
 1. [Sign up for the Watson Studio](#1-sign-up-for-the-watson-studio)
-2. [Create a new Watson Studio project](#2-create-a-new-watson-studio-project)
-3. [Create the notebook](#3-create-the-notebook)
-4. [Run the notebook](#4-run-the-notebook)
-5. [Save and Share](#5-save-and-share)
+1. [Create a new Watson Studio project](#2-create-a-new-watson-studio-project)
+1. [Create the notebook](#3-create-the-notebook)
+1. [Run the notebook](#4-run-the-notebook)
+1. [Save and Share](#5-save-and-share)
 
 ### 1. Sign up for the Watson Studio
 
 Log in or sign up for IBM's [Watson Studio](https://dataplatform.ibm.com).
 
 > Note: if you would prefer to skip the remaining Watson Studio set-up steps and just follow along by viewing the completed Notebook, simply:
-> * View the completed [notebook](https://github.com/IBM/pca-deep-dive-using-watson-studio/blob/master/notebooks/deep_dive_pca.ipynb) and its outputs, as is.
+> * View the completed [notebook](https://github.com/IBM/pca-deep-dive-using-watson-studio/blob/examples/deep_dive_pca.ipynb) and its outputs, as is.
```

Comment:
If using blob, then probably need branch name here like ".../blob/master/examples" or might be better to just use examples/deep_dive_pca.ipynb since it is relative to the repo root (and no special viewer).

Reply:
good catch - fixed